### ISSUE TITLE
CESM's rof2ocn liquid maps likely have incorrect mask_b and frac_b

### ISSUE THREAD

**Issue by billsacks** [ESMCI/cime#2010]
Due to #2009 @mnlevy1981 and I feel that we should remake all of CESM's rof2ocn liquid maps (once #2009 is fixed). For at least one case, we should then check if this changes answers at all - i.e., whether the old, buggy mask_b and frac_b had any impact on the system.

**Comment by alperaltuntas** [comment]
I generated two sets of rx1->gx37 maps: (1) using the mapping tool without the latest fixes, and (2) using the mapping tool with the latest fixes (#2007 #2013 #2022). I ran C cases with both maps and compared the results. It appears that answers do NOT change, so I think it's not necessary to remake rof2ocn liq maps. (Answer changes I saw earlier were because I have not specified the coastal grid files in nml file for the map generating tool, and so the nearest neighbor mappings were not confined to the coast.)
